provider/docker/docker.go

```diff
@@ -827,8 +827,16 @@ func (p *Provider) listServices(ctx context.Context, dockerClient client.APIClie
 	if err != nil {
 		return []dockerData{}, err
 	}
+
+	serverVersion, err := dockerClient.ServerVersion(ctx)
+
 	networkListArgs := filters.NewArgs()
-	networkListArgs.Add("scope", "swarm")
+	// https://docs.docker.com/engine/api/v1.29/#tag/Network (Docker 17.06)
+	if hasRequiredAPIVersion(serverVersion.APIVersion, "1.29", true) {
```

Comment:
Reading this, it makes me thinks it's only for `1.29`, where it's on `1.29` and above right ?

Reply:
it's >1.29